stdlib/2/_io.pyi

```diff
@@ -19,91 +19,149 @@ class _IOBase(BinaryIO):
     def fileno(self) -> int: ...
     def flush(self) -> None: ...
     def isatty(self) -> bool: ...
-    def read(self, n: int = ...) -> bytes: ...
     def readable(self) -> bool: ...
-    def readline(self, limit: int = ...) -> bytes: ...
-    def readlines(self, hint: int = ...) -> list[bytes]: ...
     def seek(self, offset: int, whence: int = ...) -> int: ...
     def seekable(self) -> bool: ...
     def tell(self) -> int: ...
     def truncate(self, size: Optional[int] = ...) -> int: ...
     def writable(self) -> bool: ...
-    def write(self, s: bytes) -> int: ...
+    def __exit__(self, type, value, traceback) -> bool: ...
+    # This just returns self:
+    def __iter__(self) -> Iterator[bytes]: ...
+    # The parameter type of writelines[s]() is determined by that of write():
     def writelines(self, lines: Iterable[bytes]) -> None: ...
+    # The return type of readline[s]() and next() is determined by that of read():
+    def readline(self, limit: int = ...) -> bytes: ...
+    def readlines(self, hint: int = ...) -> list[bytes]: ...
     def next(self) -> bytes: ...
-    def __iter__(self) -> Iterator[bytes]: ...
-    def __enter__(self) -> '_IOBase': ...
-    def __exit__(self, t: Optional[Type[BaseException]], value: Optional[BaseException],
-                 # TODO: traceback should be TracebackType but that's defined in types
-                 traceback: Optional[Any]) -> bool: ...
 
 class _BufferedIOBase(_IOBase):
-    def read1(self, n: int) -> str: ...
-    def read(self, n: int = ...) -> str: ...
+    def read1(self, size: int) -> bytes: ...
+    def read(self, size: int = ...) -> bytes: ...
     def readinto(self, buffer: bytearray) -> int: ...
-    def write(self, s: str) -> int: ...
+    def write(self, s: bytes) -> int: ...
     def detach(self) -> "_BufferedIOBase": ...
+    def __enter__(self) -> '_BufferedIOBase': ...
```

Comment:
Looks odd to have double quotes on one line and single on the next. Since this is a stub, we can just remove all quotes anyway.